tests/test_concrete_not_packed_elf64.py

```diff
@@ -27,108 +27,107 @@
-    the_sp = new_concrete_state.solver.eval(new_concrete_state.regs.sp)
-    concrete_memory = new_concrete_state.memory.load(the_sp,20)
-    assert(not concrete_memory.symbolic)
-
-    arg0 = claripy.BVS('arg0', 8*32)
-    symbolic_buffer_address = new_concrete_state.regs.rbp-0xc0
-
-    concrete_memory_2 = new_concrete_state.memory.load(symbolic_buffer_address, 36)
-    assert(not concrete_memory_2.symbolic)
-    new_concrete_state.memory.store(symbolic_buffer_address, arg0)
-
-    # We should read symbolic data from the page now
-    symbolic_memory = new_concrete_state.memory.load(symbolic_buffer_address, 36)
-    assert(symbolic_memory.symbolic)
-
-    # symbolic exploration
-    simgr = p.factory.simgr(new_concrete_state)
-    find_addr=DROP_STAGE2_V2
-    avoid_addrs=[DROP_STAGE2_V1, VENV_DETECTED, FAKE_CC]
-
-    simgr.use_technique(angr.exploration_techniques.DFS())
-    simgr.use_technique(angr.exploration_techniques.Explorer(find=find_addr, avoid=avoid_addrs))
-
-    new_concrete_state.globals["hit_malloc_sim_proc"] = False
-    new_concrete_state.globals["hit_memcpy_sim_proc"] = False
-
-    def check_hooked_simproc(state):
-        sim_proc_name = state.inspect.simprocedure_name
-        if sim_proc_name == "malloc":
-            state.globals["hit_malloc_sim_proc"] = True
-        elif sim_proc_name == "memcpy":
-            state.globals["hit_memcpy_sim_proc"] = True
-
-    new_concrete_state.inspect.b('simprocedure', action=check_hooked_simproc)
-    simgr.explore()
-
-    new_symbolic_state = simgr.stashes['found'][0]
-
-    # Assert we hit the re-hooked SimProc.
-    assert(new_symbolic_state.globals["hit_malloc_sim_proc"])
-    assert(new_symbolic_state.globals["hit_memcpy_sim_proc"])
-
-    binary_configuration = new_symbolic_state.solver.eval(arg0, cast_to=int)
-
-    new_concrete_state = execute_concretly(p, new_symbolic_state, DROP_STAGE2_V2, [(symbolic_buffer_address, arg0)], [])
-
-    # Asserting we reach the dropping of stage 2
-    nose.tools.assert_true(new_concrete_state.solver.eval(new_concrete_state.regs.pc) == DROP_STAGE2_V2)
-
-    # Go to the end.
-    new_concrete_state = execute_concretly(p, new_concrete_state, BINARY_EXECUTION_END, [], [])
-    nose.tools.assert_true(new_concrete_state.solver.eval(new_concrete_state.regs.pc) == BINARY_EXECUTION_END)
-
-def run_all():
-    functions = globals()
-    all_functions = dict(filter((lambda kv: kv[0].startswith('test_')), functions.items()))
-    for f in sorted(all_functions.keys()):
-        if hasattr(all_functions[f], '__call__'):
-            if hasattr(all_functions[f], 'setup'):
-                all_functions[f].setup()
-            try:
-                all_functions[f]()
-            finally:
-                if hasattr(all_functions[f], 'teardown'):
-                    all_functions[f].teardown()
+class testConcrete(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
This class only has one test method, but if it had more I think this should use `setUp`/`tearDown` so there is a fresh server for each test.